include/config/keybind.h

```diff
@@ -11,8 +11,8 @@ struct keybind {
 	uint32_t modifiers;
 	xkb_keysym_t *keysyms;
 	size_t keysyms_len;
-	struct wl_list actions;
-	struct wl_list link;
+	struct wl_list actions;  /* action::link */
```

Comment:
We ought to stick with either `::` or `.`
I think I saw a change in wlroots recently moving from `::` to `.` which I guess more C like. I don't really mind which, but consistency is good. (The `.` was used in action.h above)

Reply:
woops. Saw the wlroots change as well but for some reason still used `::` here.